opae.cfg

```diff
@@ -0,0 +1,459 @@
+            "configuration": {}
+          }
+        ],
+        "fpgainfo": [
+          {
+            "enabled": true,
+            "module": "libboard_n3000.so",
+            "devices": [
+              { "device": "n3000_pf", "feature_id": "*" },
+              { "device": "n3000_vf", "feature_id": "*" }
+            ]
+          }
+        ],
+        "fpgad": [
+          {
+            "enabled": true,
+            "module": "libfpgad-vc.so",
+            "devices": [ "n3000_pf" ],
+            "configuration": {
+              "cool-down": 30,
+              "get-aer":     [ "setpci -s %s ECAP_AER+0x08.L",
+                               "setpci -s %s ECAP_AER+0x14.L" ],
+              "disable-aer": [ "setpci -s %s ECAP_AER+0x08.L=0xffffffff",
+                               "setpci -s %s ECAP_AER+0x14.L=0xffffffff" ],
+              "set-aer":     [ "setpci -s %s ECAP_AER+0x08.L=0x%08x",
+                               "setpci -s %s ECAP_AER+0x14.L=0x%08x" ],
+              "sensor-overrides": [
+                {
+                  "enabled": true,
+                  "name": "12V AUX Voltage",
+                  "low-warn": 11.40,
+                  "low-fatal": 10.56
+                }
+              ]
+            }
+          }
+        ],
+        "rsu": []
+      }
+    },
+
+    "n5010": {
+      "enabled": true,
+      "platform": "Silicom FPGA SmartNIC N5010 Series",
+
+      "devices": [
+        { "name": "n5010_pf", "id": [ "0x1c2c", "0x1000", "0", "0" ] },
+        { "name": "n5010_vf", "id": [ "0x1c2c", "0x1001", "0", "0" ] }
+      ],
+
+      "opae": {
+        "plugin": [
+          {
+            "enabled": true,
+            "module": "libxfpga.so",
+            "devices": [ "n5010_pf", "n5010_vf" ],
+            "configuration": {}
+          }
+        ],
+        "fpgainfo": [
+          {
+            "enabled": true,
+            "module": "libboard_n5010.so",
+            "devices": [
+              { "device": "n5010_pf", "feature_id": "0xe" },
+              { "device": "n5010_vf", "feature_id": "0xe" }
+            ]
+          }
+        ],
+        "fpgad": [
+          {
+            "enabled": true,
+            "module": "libfpgad-vc.so",
+            "devices": [ "n5010_pf" ],
+            "configuration": {
+              "cool-down": 30,
+              "get-aer":     [ "setpci -s %s ECAP_AER+0x08.L",
+                               "setpci -s %s ECAP_AER+0x14.L" ],
+              "disable-aer": [ "setpci -s %s ECAP_AER+0x08.L=0xffffffff",
+                               "setpci -s %s ECAP_AER+0x14.L=0xffffffff" ],
+              "set-aer":     [ "setpci -s %s ECAP_AER+0x08.L=0x%08x",
+                               "setpci -s %s ECAP_AER+0x14.L=0x%08x" ],
+              "sensor-overrides": []
+            }
+          }
+        ],
+        "rsu": []
+      }
+    },
+
+    "n6000": {
+      "enabled": true,
+      "platform": "Intel Acceleration Development Platform N6000",
+
+      "devices": [
```

Comment:
Tom(Redhat) added  N6000 with out sub device id 
https://github.com/OPAE/opae-sdk/pull/2675

Beta1 release doesn't support  sub device id 

can you add to config file?

Reply:
It's actually there in the "ofs" section below.